app/pages/home/home.tsx

```diff
@@ -1,83 +1,107 @@
 import { Spinner } from '@blockstack/ui';
+import { useHotkeys } from 'react-hotkeys-hook';
 
-import { RootState } from '../../store';
-import { getAddressTransactions } from '../../store/transaction/transaction.actions';
-import { openInExplorer } from '../../utils/explorer';
-import { selectAddress } from '../../store/keys/keys.reducer';
-import { getAddressDetails } from '../../store/address/address.actions';
-import { selectAddressBalance } from '../../store/address/address.reducer';
-import { selectTransactions } from '../../store/transaction/transaction.reducer';
-import { selectPendingTransactions } from '../../store/pending-transaction/pending-transaction.reducer';
+import { RootState } from '@store/index';
+import { openInExplorer } from '@utils/external-links';
+import { selectAddress } from '@store/keys';
+import { selectAddressBalance } from '@store/address';
 import {
-  homeActions,
-  selectTxModalOpen,
-  selectReceiveModalOpen,
-} from '../../store/home/home.reducer';
+  selectTransactionList,
+  selectTransactionsLoading,
+  selectTransactionListFetchError,
+} from '@store/transaction';
+import { selectPendingTransactions } from '@store/pending-transaction';
+import { homeActions, selectTxModalOpen, selectReceiveModalOpen } from '@store/home';
 import {
   TransactionList,
   StackingPromoCard,
   StackingRewardCard,
   TransactionListItem,
   BalanceCard,
-} from '../../components/home';
-import { TransactionModal } from '../../modals/transaction/transaction-modal';
-import { ReceiveStxModal } from '../../modals/receive-stx/receive-stx-modal';
-import { useInterval } from '../../hooks/use-interval';
-import { TransactionListItemPending } from '../../components/home/transaction-list/transaction-list-item-pending';
-import { pendingTransactionSuccessful } from '../../store/transaction/transaction.actions';
+} from '@components/home';
+import { TransactionModal } from '@modals/transaction/transaction-modal';
+import { ReceiveStxModal } from '@modals/receive-stx/receive-stx-modal';
+import { TransactionListItemPending } from '@components/home/transaction-list/transaction-list-item-pending';
+
 import { Api } from '../../api/api';
-import { safelyFormatHexTxid } from '../../utils/safe-handle-txid';
-import { safeAwait } from '../../utils/safe-await';
+import { increment, decrement } from '@utils/mutate-numbers';
+import { selectActiveNodeApi } from '@store/stacks-node';
 import { HomeLayout } from './home-layout';
 
 export const Home: FC = () => {
   const dispatch = useDispatch();
-  const { address, balance, txs, pendingTxs, txModalOpen, receiveModalOpen } = useSelector(
-    (state: RootState) => ({
-      address: selectAddress(state),
-      txs: selectTransactions(state),
-      balance: selectAddressBalance(state),
-      pendingTxs: selectPendingTransactions(state),
-      txModalOpen: selectTxModalOpen(state),
-      receiveModalOpen: selectReceiveModalOpen(state),
-    })
-  );
+  const {
```

Comment:
i personally like to make these kinds of things into hooks, such as `useAppState`